src/FakeItEasy.Analyzer/UnusedReturnValueAnalyzer.cs

```diff
@@ -2,21 +2,17 @@ namespace FakeItEasy.Analyzer
 {
     using System;
     using System.Collections.Immutable;
-    using System.Linq;
     using Microsoft.CodeAnalysis;
     using Microsoft.CodeAnalysis.CSharp;
     using Microsoft.CodeAnalysis.CSharp.Syntax;
     using Microsoft.CodeAnalysis.Diagnostics;
-    using static DiagnosticDefinitions;
 
     [DiagnosticAnalyzer(LanguageNames.CSharp)]
     public class UnusedReturnValueAnalyzer : DiagnosticAnalyzer
     {
-        private static readonly ImmutableDictionary<string, DiagnosticDescriptor> Diagnostics =
-            GetDiagnosticsMap(nameof(UnusedCallSpecification));
+        private static readonly ImmutableDictionary<string, DiagnosticDescriptor> DiagnosticsMap = CreateDiagnosticsMap();
 
-        public override ImmutableArray<DiagnosticDescriptor> SupportedDiagnostics { get; } =
-            Diagnostics.Values.ToImmutableArray();
+        public override ImmutableArray<DiagnosticDescriptor> SupportedDiagnostics { get; } = ImmutableArray.Create(DiagnosticDefinitions.UnusedCallSpecification);

```

Comment:
Don't know why I removed it, actually... the line is certainly long enough to warrant it. I'll put it back, and since I'm at it, I'll also make the change suggested by @blairconrad.